ceph/ceph_admin/bootstrap.py

```diff
@@ -179,9 +179,22 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         args = config.get("args")
         custom_repo = args.pop("custom_repo", None)
         custom_image = args.pop("custom_image", True)
+        build = config.get("build")
+        container_image = config.get("container_image")
+        base_url = config.get("base_url")
+
+        if build == "released" or custom_repo.lower() == "cdn":
+            custom_repo = "cdn"
+            custom_image = False
+        elif custom_repo != "cdn":
+            custom_repo = base_url
+            custom_image = container_image
 
         if custom_repo:
```

Comment:
I think, we need to remove this part, custom_repo and custom image value is been already set above.

Reply:
@sunilkumarn417 I think we need this bcz base_url and container_image is assigned when it's passed from CLI
I think you are getting confused now
Sure will discuss on that.